core/local/atom/producer.js

```diff
@@ -115,7 +124,9 @@ module.exports = class Producer {
         }
         if (stats) scanEvent.stats = stats
         if (incomplete) scanEvent.incomplete = incomplete
-        entries.push(scanEvent)
+        if (!isIgnored(scanEvent, this.ignore)) {
```

Comment:
I wonder if we could have situations where we want to ignore some directories for which we can't get the `stats` and thus won't match against the rule.

Reply:
Wouldn't the issue be the same in the `add_infos` step anyway?

Reply:
Probably. This would be a separate issue then